integration-tests/src/test/scala/chiselTests/util/experimental/algorithm/Bitwise.scala

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import chisel3._
+import chisel3.util._
+import chisel3.experimental.util.algorithm._
+import chiseltest._
+import chiseltest.formal._
+import org.scalatest.flatspec.AnyFlatSpec
+import scala.math.min
+
+// Copied from rocket-core
```

Comment:
what is `rocket-core`? Can you provide a better link, and are there any license implications to "copied from"?